docs/git-workflow.md

```diff
@@ -25,9 +25,25 @@ The Jetpack repo uses the following "reserved" branch name conventions:
 
 ## Keeping Your Branch Up To Date
 
-While it is tempting to merge from `master` into your branch frequently, this leads to a messy history because each merge creates a merge commit. When working by yourself, it is best to use `git pull --rebase master`, but if you're pushing to a shared repo, it is best to not do any merging or rebasing until the feature is ready for final testing, and then do a [rebase](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) at the very end. This is one reason why it is important to open pull requests whenever you have working code.
-
-If you have a Pull Request branch that cannot be merged into `master` due to a conflict (this can happen for long-running Pull Request discussions), it's still best to rebase the branch (rather than merge) and resolve any conflicts on your local copy before updating the Pull Request with `git push --force`. **Be aware** that this will **replace** any commits currently in your shared branch, so anyone who is also using that branch will be in trouble. Only use `git push --force` if the Pull Request is ready to merge and no one else is using it (or of you have coordinated the force-push with the other developers working on the branch).
+There are two ways to update your branch with changes to `master` or a parent PR.
+
+1. **Merge:** After a `git fetch`, execute `git merge origin/master` to pull in the latest changes. If there are conflicts, you'll need to resolve them by hand and then commit manually.
```

Comment:
```suggestion
1. **Merge:** After a `git fetch`, execute `git merge origin/master` to pull in the latest changes. If there are conflicts, you'll need to resolve them by hand and then commit manually.
**Preferred for branches already pushed to GitHub.**
```

Reply:
I wonder if the extra bolding in this suggestion and the next is a bit much. We already say what's preferred after listing the options.

Reply:
I'm open to other thoughts.

My thinking was try to make it foolproof obvious when to use each, but if others agree it is overkill, not the hill I'm going to die on.

Reply:
Same here, if others think the additional bolded statements would be useful I'll go along with it.

Reply:
@jeherve care to be the tie-breaker for us on this one?

Reply:
Hmm. Still kind of a lot of bolding, but at least it's all in one place.